go.mod

```diff
@@ -1,17 +1,19 @@
 module github.com/openshift/windows-machine-config-operator
 
-go 1.20
+go 1.23
+
+toolchain go1.23.0
```

Comment:
please revert

Reply:
hmm... reverting this sets it to 1.20.12, which is causing the tests to fail. 

 error: GIMME_GO_VERSION not recognized as valid
  got: v1.20.12
Detected go version: go version go1.20.12 linux/amd64.
Kubernetes requires go1.22 or greater.
Please install go1.22 or later.